src/main/java/net/kaczmarzyk/spring/data/jpa/web/annotation/Spec.java

```diff
@@ -34,23 +33,16 @@
 
     String[] params() default {};
 
-    /**
-     * Attribute determines where to get the parameter values from
-     */
-    ParamType paramType() default ParamType.QUERY;
-
     /**
      * Character used to param delimitation. Param delimitation is skipped when separator has a 0 value.
      */
     char paramSeparator() default 0;
 
-    /**
-     * @deprecated Use {@link #params()} and {@link #paramType()} with value {@link ParamType#PATH}
-     */
-    @Deprecated
     String[] pathVars() default {};
 
     String[] headers() default {};
+
+    String[] jsonPath() default {};
```

Comment:
hmm, all others are plural, so perhaps it makes sense to use "jsonPaths" here as well (?)